<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>390</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>403</width>
    <height>390</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>403</width>
    <height>390</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Важное/Университет/НИР/Рабочая часть/LiveUSB/icons/options.png</normaloff>../Важное/Университет/НИР/Рабочая часть/LiveUSB/icons/options.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QTabWidget" name="tabSectionSettings">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabRemoteDesktop">
      <attribute name="title">
       <string>Прикладные сервисы</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTabWidget" name="tabProtocolForConnection">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>210</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="tabRDP">
          <attribute name="title">
           <string>RDP</string>
          </attribute>
          <widget class="QLineEdit" name="_edUsernameRDP">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>20</y>
             <width>321</width>
             <height>31</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="maxLength">
            <number>15</number>
           </property>
           <property name="placeholderText">
            <string>Имя пользователя</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="_edIpAddressRDP">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>80</y>
             <width>171</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="inputMask">
            <string>999.999.999.999</string>
           </property>
           <property name="text">
            <string>192.168.000.001</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
          <widget class="QLineEdit" name="_edPasswordRDP">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>50</y>
             <width>321</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="maxLength">
            <number>15</number>
           </property>
           <property name="placeholderText">
            <string>Пароль</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="_chboxUseDefaultPortRDP">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>120</y>
             <width>271</width>
             <height>24</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Использовать порт по умолчанию</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLineEdit" name="_edPortRDP">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>290</x>
             <y>110</y>
             <width>51</width>
             <height>41</height>
            </rect>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="text">
            <string>3389</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="_lbIpAddrETH_3">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>80</y>
             <width>71</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>IP-адрес:</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="tabSSH">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <attribute name="title">
           <string>SSH</string>
          </attribute>
          <widget class="QLineEdit" name="_edUsernameSSH">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>20</y>
             <width>321</width>
             <height>31</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="maxLength">
            <number>15</number>
           </property>
           <property name="placeholderText">
            <string>Имя пользователя</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="_edIpAddressSSH">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>50</y>
             <width>171</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="inputMask">
            <string>999.999.999.999</string>
           </property>
           <property name="text">
            <string>192.168.000.001</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string>IP-адрес</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="_chboxUseDefaultPortSSH">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>90</y>
             <width>271</width>
             <height>24</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Использовать порт по умолчанию</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLineEdit" name="_edPortSSH">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>290</x>
             <y>80</y>
             <width>51</width>
             <height>41</height>
            </rect>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="text">
            <string>22</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="_lbIpAddrETH_2">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>50</y>
             <width>71</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>IP-адрес:</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="_chboxUseDomainSSH">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>140</y>
             <width>101</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Исп. домен</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QLineEdit" name="_edDomainSSH">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>140</y>
             <width>171</width>
             <height>31</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="maxLength">
            <number>15</number>
           </property>
           <property name="placeholderText">
            <string>Домен</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="ltNavigation">
         <item>
          <widget class="QPushButton" name="_btCancel">
           <property name="text">
            <string>Отменить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="_btSave">
           <property name="text">
            <string>Сохранить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="_btApply">
           <property name="text">
            <string>Применить</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabNetwork">
      <attribute name="title">
       <string>Сетевые интерфейсы</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTabWidget" name="tabConnectionType">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>290</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>290</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tabWiFi">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <attribute name="title">
           <string>Wi-Fi</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="_btScanWifiNetworks">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Поиск доступных сетей Wi-Fi</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="_tableWifiNetworks">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <attribute name="horizontalHeaderCascadingSectionResizes">
              <bool>true</bool>
             </attribute>
             <attribute name="horizontalHeaderMinimumSectionSize">
              <number>57</number>
             </attribute>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>145</number>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Имя сети</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Свойство сети</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="_btConnect">
         <property name="text">
          <string>Подключиться</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>VPN</string>
      </attribute>
      <widget class="QPushButton" name="_btUseVPN">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Вкл</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="_btChooseFile">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>134</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Выбрать файл</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="_edFilePath">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>321</width>
         <height>31</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
